pkg/runtime/generate_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 GM Cruise LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestGenerate(t *testing.T) {
+	testdataPath := "../../testdata/generator/"
```

Comment:
you can also create a `testdata` folder in this runtime package so we know these files are for this package specifically and spare us the crazy relative path math
https://dave.cheney.net/2016/05/10/test-fixtures-in-go